guide/daily-spending-wallet/backup-and-recovery/cloud-backup.md

```diff
@@ -115,37 +128,56 @@ height = 541
+As mentioned in the previous section, this wallet prompts the user to backup during [First Use](https://bitcoin.design/guide/daily-spending-wallet/first-use/) and, if they opt-out, reminds them again once they have received funds.
 
-On a similar note, it’s crucial that you also explain that if someone else knows their cloud account information, they can gain access to their funds, too.
+The wallet explains to the user that their backup is stored on the cloud, and that they can regain access to their funds on another device using their respective cloud account and bitcoin wallet app. The app also explains that the backup is encrypted with their PIN and requires the user to verify the PIN by entering it a second time.
 
 {% include tip/open.html color="green" icon="check" label="Do: Inform your Users" %}
 
-Before creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation because there will likely be a few seconds between when a user clicks to upload to the cloud provider and when it completes.
+Before creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. Let them know where in their cloud storage they can find their backup, and give the file an understandable name. For example, a filename like `BitcoinWalletBackup_Samantha` conveys far more meaning than a randomly generated number.
 
 {% include tip/close.html %}
 
-### Tradeoffs
+## Error Handling
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/automatic-cloud-backup/cloud-backup-error.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/automatic-cloud-backup/cloud-backup-error@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/automatic-cloud-backup/cloud-backup-error@2x.png"
+   alt-text = "Screen showing an error message for a failed auto cloud backup"
+   caption = "This wallet notifies the user when there is a backup error and provides next steps"
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+In normal conditions, this wallet performs an automatic cloud backup after sending or receiving any payment. However, there could be situations where this fails to happen, such as an error or outage on the side of the cloud provider. Regardless of the reason, it's good to notify the user if there is a problem. This wallet offers a list of common issues and solutions, and also offers the ability to submit a support request.
```

Comment:
Instead of a support request, maybe first send user to the settings of their cloud provider and ensure their account is still working as it should?

I'd also recommend sentence-casing button labels.

Reply:
Updated. I sentence-cased the buttons. For sending the user to their cloud provider, I think that would fit under "view common issues". I updated the text to drive people to that first, and then a support request if that doesn't work.